centreon/www/front_src/src/CloudNotificationsConfiguration/EditPanel/FormInputs/useFormInputs.tsx

```diff
@@ -48,18 +48,26 @@ const useFormInputs = ({ panelWidth }: Props): object => {
   const { classes } = useStyles({ isExtraFieldHidden });
   const { t } = useTranslation();
 
+  const titleAttributes = {
+    classes: { root: classes.titleGroup },
+    variant: Variant.subtitle1
```

Comment:
Btw, you can use the "Variant" from MUI: https://github.com/mui/material-ui/blob/master/packages/mui-material/src/styles/createTypography.d.ts#L5-L18